hazelcast-sql/src/main/java/com/hazelcast/sql/impl/calcite/opt/physical/MapScanPhysicalRule.java

```diff
@@ -67,24 +62,6 @@ public void onMatch(RelOptRuleCall call) {
             transforms.add(mapScan);
         }
 
-        // Try adding index scans.
```

Comment:
Why removing just part of it? Is it used?

Reply:
I am removing IMDG engine step by step to reduce count of removed LOC in each PR. 
About this part : the change allowed me to delete enough files, but not so many.
I won't expect you like to review a giant PRs.

Reply:
So why touching this file at all?

Reply:
Just because I know that if I remove index creation and move/remove all related files - the changeset size would be appropriate. No tons of removed files.